lib/ecto/postgres/pg_rational_migrations.ex

```diff
@@ -463,6 +486,42 @@ defpgmodule Vtc.Ecto.Postgres.PgRational.Migrations do
     :ok
   end
 
+  @doc section: :migrations_functions
+  @doc """
+  Creates Creates `rational_private.modulo(a, b)` backing function for the `%` operator
```

Comment:
```suggestion
Creates `rational_private.modulo(a, b)` backing function for the `%` operator
```

Reply:
In 2e861d2c78616a6ce016e766397f97c850a28a01